packages/elements/tsconfig.json

```diff
@@ -10,7 +10,9 @@
     "jsx": "preserve",
     "jsxImportSource": "solid-js",
     "resolveJsonModule": true,
+    "skipLibCheck": true,
```

Comment:
you might want to turn this back on

Reply:
I agree this isn't 100% ideal, but I'm getting super unrelated node type errors that are blocking. let's just keep in mind that we want to switch these back to false at some point